schema/ont_papers.ttl

```diff
@@ -0,0 +1,68 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix fabio: <http://purl.org/spar/fabio/> .
+@base <http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/> .
+
+<https://w3id.org/ocs/ont/papers/> rdf:type owl:Ontology ;
+                                    rdfs:comment "This ontology describes properties used in OpenCS knowledge graph about research articles from the domain of Computer Science." .
+
+#################################################################
+#    Object Properties
+#################################################################
+
+###  http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/hasRelatedPapers
+:hasRelatedPapers rdf:type owl:ObjectProperty ,
+                           owl:SymmetricProperty ,
+                           owl:ReflexiveProperty ;
+                  rdfs:domain fabio:ResearchPaper ;
+                  rdfs:range fabio:ResearchPaper ;
+                  rdfs:comment "Indicates that a research paper has similar content to another research paper, determined by the cosine similarity of their embeddings." .
+
+
+###  http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/hasRelatedTopics
+:hasRelatedTopics rdf:type owl:ObjectProperty ,
+                           owl:IrreflexiveProperty ;
+                  rdfs:domain fabio:ResearchPaper ;
+                  rdfs:comment "Specifies a list of topics or concepts that are related to the given research paper." .
+
+
+#################################################################
+#    Data properties
+#################################################################
+
+###  http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/hasWordEmbedding
+:hasWordEmbedding rdf:type owl:DatatypeProperty ,
+                           owl:FunctionalProperty ;
+                  rdfs:domain fabio:ResearchPaper ;
+                  rdfs:range xsd:string ;
+                  rdfs:comment "Represents the embedded vector of the title and abstract of a given research paper." .
+
+
+###  http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/relationScore
+:relationScore rdf:type owl:DatatypeProperty ,
+                        owl:FunctionalProperty ;
+               rdfs:range xsd:float ;
+               rdfs:comment "Quantifies the degree of relationship between a given research paper and a topic. The higher the value, the stronger the association, with no fixed range specified." .
+
+
+###  http://www.semanticweb.org/tomas/ontologies/2023/10/untitled-ontology-3/similarityScore
+:similarityScore rdf:type owl:DatatypeProperty ,
+                          owl:FunctionalProperty ;
+                 rdfs:domain fabio:ResearchPaper ;
```

Comment:
Something is wrong here. If I remember correctly, you attached this property not to the paper itself, but to some helper instance, one instance per relation. I think you need something like `RelatedPaperRelation` class to represent this.

Note that you don't have to specify this class explicitly in the KG – you can just model it in the ontology and we will know it's there.

Reply:
Oh, fixing this will require more changes to the ontology elsewhere. :)

Reply:
I think there is a similar problem with the related topics relation, I am attaching a possible solution for both, is it ok?
![relations](https://github.com/OpenCS-ontology/OpenCS/assets/72312243/3301f84d-5900-42d8-a06e-87473865dd37)

Reply:
Looks better :)

Sugestions:
- Make a superclass (let's say `:PaperRelation`) for `:RelatedPaperRelation` and `:RelatedTopicRelation`. This way you could define the range of :hasRelation more easily (as just :PaperRelation)
- Merge the two props for scores into one, they do the same thing, right?
- For relation target – leave the range empty for now, I will show you on Friday how to make an axiom to express the range in a situation like this. For us to be able to discuss it, please send me a draft including at least these changes today, ok?